### ISSUE TITLE
Request to add Mapbox's Landsat Live base

### ISSUE THREAD

**Issue by makella** [nationalparkservice/npmap-builder#208]
From @ryanjhodge:

<em>I think I&M folks would find this basemap useful/exciting, they are forever chasing the latest imagery....</em>
https://www.mapbox.com/blog/landsat-live-live/

**Comment by nateirwin** [comment]
I have an email out to Mapbox support about this, and hopefully we'll hear back soon. Once we have access, I'll get it added to Builder!

**Comment by ryanjhodge** [comment]
Did they ever get back to you? Was hoping to show this off next week.

**Comment by nateirwin** [comment]
Unfortunately, no. Let me try again.

**Comment by ryanjhodge** [comment]
If you would like to use this imagery via your Mapbox account, send us an e-mail at satellite@mapbox.com to get an early preview.

**Comment by nateirwin** [comment]
Yep, I've sent them two emails now. I'll let you know when I hear back.

**Comment by thisishap** [comment]
L.mapbox.accessToken = 'pk.eyJ1IjoibXVsbG92ZXJpdCIsImEiOiIzVzZyY05JIn0.QxNcdiemvp44I0vi0ppBuw';
var map = L.mapbox.map('map', 'mapbox.landsat-live-32,pdgoodman.2e9b9266', {
        minZoom: 10,
        maxZoom: 13
    }).setView([37.888888889, -121.23600370904541], 10).addControl(L.mapbox.geocoderControl('mapbox.places'));

**Comment by nateirwin** [comment]
Thanks, @thisishap, I just added this to NPMap.js as a preset: https://github.com/nationalparkservice/npmap.js/commit/faa20dd6ae598d271a2b6fc4e263b84725116eb6. I'll roll this into Builder early next week.

**Comment by nateirwin** [comment]
Done.
